.circleci/config.yml

```diff
@@ -373,9 +373,11 @@ jobs:
         run:
           name: Early return if this build is running on upstream
           command: |
-            if [ "$CIRCLE_PR_NUMBER" ]; then
-              echo "Build on upstream"
+            if [ -n "$CIRCLE_PR_NUMBER" ]; then
+              echo "Build on fork"
+            else
               circleci step halt
+              echo "Build on upstream"
```

Comment:
I think `circleci step halt` will immediately exit, so we'll need to put this message before the halt.

And it looks like modern invocation is `circleci-agent` vs. `circleci` but I don't understand the difference. https://circleci.com/docs/2.0/configuration-reference/#ending-a-job-from-within-a-step